### DISCUSSION TITLE
regional_epinow and rt_opts parameters

### DISCUSSION THREAD

**Discussion by AndrewRiceMGW** [epiforecasts/EpiNow2#295] [General]
Hi is there a way to run regional_epinow and specify the pop parameter of rt_ops for the susceptibility number for each country?

**Comment by seabbs** [comment]
Hi @AndrewRiceMGW, 

Yes you should be able to adapt the entry in the `rt_opts` list. An example of doing that is here: https://github.com/epiforecasts/europe-covid-forecast/blob/d5b4f350a3705dec75470352d0fb71c8ce491d6c/rt-forecast/case.R#L40

not the most seem-less interface and the assumption you would need to make for vaccination is of homogeneous mixing with no change in susceptibility over time. This is something we are looking at for the next version (#234)
